### DISCUSSION TITLE
when program is full screen, scroll cannot work

### DISCUSSION THREAD

**Discussion by Freed-Wu** [zellij-org/zellij#3603] [Q&A]
I comes from tmux and found some different behaviour.

When I open a full screen program like (n)vim, then enter scroll mode, press `j/k` cannot scroll up/down.

Is it an expected behaviour?

**Comment by Lockszmith-GH** [comment]
> I comes from tmux and found some different behaviour.

Don't start with this, you'll find the community isn't fond of this comparison.

That being said, I don't know whether it's expected behavior or not, but please share:
- what are your expectations?
- what OS are you running
- which terminal emulator (and if a remote connection, which OS the terminal emulator is running on)
- In your specific case, which version of nvim?

These all play a factor, and at least I'll know if I'm testing on my end (Debian, from Windows Terminal on Windows 11) produces similar results or not.

**Reply by Freed-Wu** [reply]
Sorry for my sentences.

For example, I run some commands in shell, then run `nvim`, in tmux, enter scroll mode, press `u` I can see the results of the commands in shell, in zellij, press `u` will do nothing.

I don't mean which behaviour is better. In fact, press `<C-Z>` to suspend nvim then enter scroll mode of zellij, press `u` can work. So it is not an important problem. If someone can share some reason about this behaviour, it will be very thanksful.

Sorry for my sentences again.

nvim: 0.10.0
OS: ArchLinux rolling, NixOS unstable

**Reply by Lockszmith-GH** [reply]
The default keybinding for Zellij use a different flow than tmux/screen, however, if you want to mimic that, you can edit the config.kdl and make it work the way you like.

**Reply by Freed-Wu** [reply]
When I run `minicom` to communicate with another serial device, I can scroll to see previous message. however, zellij cannot work :cry:
